Podcast/SeriesDetailViewController.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  SeriesDetailViewController.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 2/13/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class SeriesDetailViewController: UIViewController, SeriesDetailHeaderViewDelegate, UITableViewDelegate, UITableViewDataSource, EpisodeTableViewCellDelegate {
+    
+    let seriesHeaderHeight: CGFloat = SeriesDetailHeaderView.height
+    
+    let sectionHeaderHeight: CGFloat = 64.0
+    let sectionTitleY: CGFloat = 32.0
+    let sectionTitleH: CGFloat = 18.0
+    let padding: CGFloat = 18.0
+    let sepH: CGFloat = 1.0
+    
+    var seriesHeaderView: SeriesDetailHeaderView!
+    var epsiodeTableView: UITableView!
+    
+    var series: Series? {
+        didSet {
+            updateViewData()
```

Comment:
Don't use a didset to call a function, if you want a function to get called, have the controller call it